CharlieBackend.Root/CompositionRoot.cs

```diff
@@ -54,6 +54,8 @@ public static void InjectDependencies(IServiceCollection services, IConfiguratio
             services.AddScoped<IScheduleService, ScheduleService>();
             services.AddScoped<INotificationService, NotificationService>();
             services.AddScoped<IAttachmentService, AttachmentService>();
+            services.AddScoped<IHometaskRepository, HometaskRepository>();
```

Comment:
we don't need Repository here, cuz we inject it in unitOfWork and always use it like this:
`_unitOfWork.AttachmentRepository.GetAttachmentsByIdsAsync(createHometaskDto.AttachmentIds);`